apps/src/lib/levelbuilder/lesson-editor/AddResourceDialog.jsx

```diff
@@ -123,17 +125,22 @@ export default class AddResourceDialog extends Component {
         headers: {'X-CSRF-Token': formData.get('authenticity_token')},
         body: formData
       })
+        .then(response => {
```

Comment:
In the process of this PR, I found out that this error handling was broken so I fixed it.